Source/Scene/ScreenSpaceCameraController.js

```diff
@@ -136,6 +136,12 @@ function ScreenSpaceCameraController(scene) {
    * @default {@link Number.POSITIVE_INFINITY}
    */
   this.maximumZoomDistance = Number.POSITIVE_INFINITY;
+  /**
+   * A multiplier for the speed at which the camera will zoom.
+   * @type {Number}
+   * @default 5.0
+   */
+  this.zoomFactor = 5.0;
```

Comment:
I'd like to also set the private attribute here `this._zoomFactor = this.zoomFactor` and reset the usages back to the private member to make it backward compatible.  I know I'm using the private member now and other users might be doing the same.  I see this pattern elsewhere in this file.